.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I overlooked this too.
I don't like this either.
I don't like forcing commit message conventions for the local commits.
Same reason as here https://github.com/anonklub/anonklub/pull/341#discussion_r1481053692: it is a dev branch, I don't care to see dirty commits. I even find annoying to force me coming up with clean names for all my commits on my local dev branch. It is killing my flow.
If you need clear commit messages in your own branch to make sense of what you are doing... I am arguing your PR/branch is too big and you should split your work in a smaller PR.

My I DO want/like is conventional commit messages for the merge commit (squash and merge). The commit message on `main` MUST be clean of course.


So it would be nice to define a github action instead that checks for the PR titlle.